test/local/db.js

```diff
@@ -270,92 +261,109 @@ describe('redis enabled', () => {
     return db.devices('wibble')
       .then(() => {
         assert.equal(pool.get.callCount, 1)
-        assert.equal(redis.getAsync.callCount, 1)
+        assert.equal(redis.get.callCount, 1)
+        assert.equal(redis.get.args[0].length, 1)
+        assert.equal(redis.get.args[0][0], 'wibble')
       })
   })
 
-  describe('redis error:', () => {
-    beforeEach(() => redis.on.args[0][1]({ message: 'foo', stack: 'bar' }))
-
-    it('should log the error', () => {
-      assert.equal(log.error.callCount, 1, 'log.error was called once')
-      assert.equal(log.error.args[0].length, 1, 'log.error was passed one argument')
-      assert.deepEqual(log.error.args[0][0], {
-        op: 'db.redis.error',
-        err: 'foo',
-        stack: 'bar'
-      }, 'log.error was passed the error details')
-    })
+  it('should call redisConnection.get in db.sessions', () => {
+    return db.sessions('wibble')
+      .then(() => {
+        assert.equal(redis.get.callCount, 1)
+        assert.equal(redis.get.args[0].length, 1)
+        assert.equal(redis.get.args[0][0], 'wibble')
+      })
   })
 
-  describe('redis.get error:', () => {
-    beforeEach(() => redis.getAsync = sinon.spy(() => P.reject({ message: 'wibble' })))
-
-    it('db.sessions should log the error', () => {
-      return db.sessions('blee')
-        .then(() => {
-          assert.equal(redis.getAsync.callCount, 1, 'redis.get was called once')
-          assert.equal(redis.getAsync.args[0].length, 1, 'redis.get was passed one argument')
-          assert.equal(redis.getAsync.args[0][0], 'blee', 'redis.get was passed uid')
-
-          assert.equal(log.error.callCount, 1, 'log.error was called once')
-          assert.equal(log.error.args[0].length, 1, 'log.error was passed one argument')
-          assert.deepEqual(log.error.args[0][0], {
-            op: 'db.redis.get.error',
-            method: 'sessions',
-            err: 'wibble'
-          }, 'log.error was passed the error details')
-        })
+  it('should call redisConnection.del in db.deleteAccount', () => {
+    return db.deleteAccount({ uid: 'wibble' })
+      .then(() => {
+        assert.equal(redis.del.callCount, 1)
+        assert.equal(redis.del.args[0].length, 1)
+        assert.equal(redis.del.args[0][0], 'wibble')
+      })
+  })
+
+  it('should call redisConnection.del in db.resetAccount', () => {
+    return db.resetAccount({ uid: 'wibble' }, {})
+      .then(() => {
+        assert.equal(redis.del.callCount, 1)
+        assert.equal(redis.del.args[0].length, 1)
+        assert.equal(redis.del.args[0][0], 'wibble')
+      })
+  })
+
+  it('should call redisPool.acquire, redisConnection.update and redisPool.release in db.updateSessionToken', () => {
+    return db.updateSessionToken({ id: 'wibble', uid: 'blee' }, P.resolve())
+      .then(() => {
+        assert.equal(acquire.callCount, 2)
```

Comment:
Is this `2` because the db acquires one for ongoing use as `this.redis`?  The tests may be slightly clearer if we reset the call count in the setup code in order to ignore that extra call to acquire.